src/platform_impl/macos/view.rs

```diff
@@ -414,13 +420,10 @@ extern "C" fn reset_cursor_rects(this: &Object, _sel: Sel) {
     } else {
       util::invisible_cursor()
     };
-
-    if !cursor.is_null() {
-      let _: () = msg_send![this,
-          addCursorRect:bounds
-          cursor:cursor
-      ];
-    }
+    let _: () = msg_send![this,
+        addCursorRect:bounds
+        cursor:cursor
+    ];
```

Comment:
why was this check removed?

Reply:
Must have been on accident when deleting my own stuff